doc/cookbook/source/examples/regression/support_vector_regression.rst

```diff
@@ -0,0 +1,47 @@
+=========================
+Support Vector Regression
+=========================
+
+Support vector regression is a regression model inspired from support vector machines. The solution can be written as:
+
+.. math::
+    f({\bf x})=\sum_{i=1}^{N} \alpha_i k({\bf x}, {\bf x_i})+b
```

Comment:
This is OK.
I would add the objective function that is minimized. You can find it in the "learning with kernels book"